src/core_plugins/kibana/server/lib/import/import_dashboards.js

```diff
@@ -0,0 +1,33 @@
+import { flatten } from 'lodash';
+import { SavedObjectsClient } from '../../../../../server/saved_objects';
+
+const acceptableTypes = [
+  'search',
+  'dashboard',
+  'visualization',
+  'index-pattern'
+];
+
+export default function importDashboards(req) {
+  const { payload } = req;
+  const config = req.server.config();
+  const force = 'force' in req.query && req.query.force !== 'false';
```

Comment:
This is my bad, force is now a boolean due to the added Joi config for this endpoint.

Should be:
`const force = 'force' in req.query && req.query.force !== false;`